pybossa/view/projects.py

```diff
@@ -1252,8 +1255,28 @@ def task_presenter(short_name, task_id):
         "user_id": user_id
     }
 
+    if task_submitter_id is not None:
```

Comment:
As user_ids  are always > 0, this could be
```python
if task_submitter_id:
```

Reply:
Will update it.